cmd/root.go

```diff
@@ -0,0 +1,42 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License
+// 2.0; you may not use this file except in compliance with the Elastic License
+// 2.0.
+
+package cmd
+
+import (
+	"sort"
+
+	"github.com/elastic/elastic-agent-changelog/internal/cobraext"
```

Comment:
Unless we plan to use `cobraext` I would not include it at this stage, as it doesn't seem to be used at the moment. We can introduce this later if needed.

Reply:
This is used in root, it provides access to the command structured, along with other functionality.

Reply:
It will also be needed for future improvements in root.go

Reply:
To my understanding `cobraext` main functionality is wrapping a standard `cobra.Command` to add `CommandContext`. I'm not sure is worth the abstraction if we don't plan to add contexts to commands. We are not going to use `ComposeCommandActions` (as discussed in https://github.com/elastic/elastic-agent-changelog-tool/pull/2#discussion_r799400217) so `cobraext` does not provide useful functionality at the moment.

Based on
> It will also be needed for future improvements in root.go
you may already think of improvements that would make sense to bring here.